presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -173,7 +174,16 @@ public PlanOptimizers(Metadata metadata, SqlParser sqlParser, FeaturesConfig fea
                         stats,
                         ImmutableSet.of(new SimplifyCountOverConstant())),
                 new WindowFilterPushDown(metadata), // This must run after PredicatePushDown and LimitPushDown so that it squashes any successive filter nodes and limits
-                new MergeWindows(),
+                new IterativeOptimizer(
+                                stats,
+                                ImmutableList.of(
+                                        new UnaliasSymbolReferences(),
+                                        new MergeWindows()),
+                                ImmutableSet.of(
```

Comment:
Add a `TODO` to add a `UnaliasSymbolReferences` once it's ported to the new iterative optimizer.